var/spack/repos/builtin/packages/perl-perlio-utf8-strict/package.py

```diff
@@ -13,3 +13,5 @@ class PerlPerlioUtf8Strict(PerlPackage):
     url = "http://search.cpan.org/CPAN/authors/id/L/LE/LEONT/PerlIO-utf8_strict-0.002.tar.gz"
 
     version("0.002", sha256="6e3163f8a2f1d276c975f21789d7a07843586d69e3e6156ffb67ef6680ceb75f")
+
+    depends_on("perl-module-build", type="build")
```

Comment:
It looks like this is a build dependency up to version 0.005. We should capture that here.
```suggestion
    depends_on("perl-module-build", type="build", when="@:0.005")
```